### PR TITLE
feat: Use jest-configurator in the Markup package

### PR DESCRIPTION
**Pull request by jimhunty**
As the title suggests.

I have also made a small code optimisation as well to take a look at.

### CODE REVIEW

File: packages/markup/__tests__/ios/markup.android.test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Text, View } from "react-native";
+import runTests from "../shared";
+import { renderTree, renderTrees } from "../../markup";
+
+describe(
+  "Markup Native tests on iOS",
```
Comment (colincclark) [inline packages/markup/__tests__/ios/markup.android.test.js]: Filename should be `ios`

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/02c356f296b90de50a3d6c094df62884cdc39765/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/e9edc0b91706688f17ce8a3fb85e36cb513801fc/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/0bdd298786d34fc668ffe77bc60bffee605c3a16/dextrose-test/index.html

**Comment by jimhunty** [comment]
@craigbilner Native and Web require different markup to be tested correctly, which is passed through in the function. We are executing a function in the other tests so its closer than maybe first thought?

The `renderTree` and `renderTrees` params were unnecessary and have been removed.

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/a0e99d6b4eca7a00c9f626f018be71262d6afe0d/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/be04dba7b06ea5370850e4460857f131988595d4/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/23fdd8ad172cf90ab2c30a4edcc77061ce805db0/dextrose-test/index.html

File: packages/markup/__tests__/web/__snapshots__/markup.web.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+        and in here
+      </span>
+    </span>
+     after it too
+  </span>
+   and here
+</div>
+`;
+
+exports[`Markup Web renders the anchor tag 1`] = `
```
Comment (craigbilner) [inline packages/markup/__tests__/web/__snapshots__/markup.web.test.js.snap]: need to tweak this test

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/53cf9a23006cecd82c091f8d0a332a58959f06ab/dextrose-test/index.html

**Pull request merged**